docs/releases.rst

```diff
@@ -4,6 +4,13 @@
     Releases
 ======================
 
+tmt-1.33
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+New section :ref:`review` describing benefits and various forms of
+pull request reviews has beed added to the :ref:`contribute` docs.
```

Comment:
`beed` looks a typo, it should be `been`?

Reply:
Yeah, good catch!

Reply:
Fixed in 2ac0eb0.

Reply:
Looks good to me :-)